cardano-cli/src/Cardano/CLI/EraBased/Commands.hs

```diff
@@ -134,6 +139,10 @@ pCmds envCli era =
         $ Opt.info (QueryCmds <$> pQueryCmds envCli)
         $ Opt.progDesc "Era-based query commands"
     , fmap StakeAddressCmds <$> pStakeAddressCmds era envCli
+    , Just
+        $ subParser "stake-pool"
+        $ Opt.info (StakePoolCmds <$> pStakePoolCmds era envCli)
+        $ Opt.progDesc "Era-based text-view commands"
```

Comment:
Copy paste error

Reply:
fixed in: https://github.com/input-output-hk/cardano-cli/pull/281